src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -213,8 +213,8 @@ - (void)startNextRead {
         // (because it's just a client problem). Use another domain and an
         // appropriately-documented code.
         [weakSelf finishWithError:[NSError errorWithDomain:kGRPCErrorDomain
-                                                      code:GRPCErrorCodeInternal
-                                                  userInfo:nil]];
+                                                      code:GRPCErrorCodeResourceExhausted
+                                                  userInfo:@{NSLocalizedDescriptionKey: @"Client out of memory."}]];
```

Comment:
Technically not out of memory, as we weren't able to allocate enough for the response (so we didn't) :) Your description in `statuscodes.md` sounds 👍 to me: "Client does not have enough memory to hold the server response."